pkg/cluster/executor/executor.go

```diff
@@ -127,3 +122,32 @@ make sure there is just one ip address in your topology fiile`, c.Host, localIP)
 		return nil, fmt.Errorf("unregistered executor: %s", etype)
 	}
 }
+
+func checkLocalIP(ip string) error {
+	ifaces, err := net.Interfaces()
+	if err != nil {
+		return errors.AddStack(err)
+	}
+
+	for _, i := range ifaces {
+		addrs, err := i.Addrs()
+		if err != nil {
+			return errors.Annotatef(err, "get address for interface %s", i.Name)
```

Comment:
I think we shouldn't return an error if we cannot get the address and continue the subsequent iteration. Because users haven't allocated IP addresses for some virtual Interfaces.